<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find nav_planner)/logging_debug.conf"/>

    <node pkg="octomap_server" type="octomap_server_node" name="octomap_server" output="screen">
		<param name="resolution" value="0.05" />
		<param name="frame_id" type="string" value="/odom" />
		<param name="sensor_model/max_range" value="5.0" />
		<param name="latch" value="true" />
		<remap from="cloud_in" to="/camera/depth/points" />
	</node>
    
    <node pkg="nav_planner" type="goal_identifier_node" name="goal_identifier" output="screen">
		<remap from="octomap" to="/octomap_full" />
        <remap from="odometry" to="/odom" />
	</node>
</launch>

